KomaMRIFiles/src/RawAcquisitionData/WriteRawData.jl

```diff
@@ -1,6 +1,6 @@
 """
-For using the Limit Struct of KomaMRICore when writing with MRIFiles
+For using the Limit Struct of KomaMRIBase when writing with MRIFiles
 """
-insertNode(xs, paramName::String, param::KomaMRICore.Limit) = begin
+insertNode(xs, paramName::String, param::KomaMRIBase.Limit) = begin
     insertNode(xs, paramName, MRIFiles.Limit(param.minimum, param.maximum, param.center))
 end
```

Comment:
Remove this.

Reply:
Addressed in branch `avoid-circular-deps`